projects/storefrontlib/cms-structure/page/page-layout/page-layout.service.ts

```diff
@@ -159,21 +169,21 @@ export class PageLayoutService implements OnDestroy {
     configAttribute: string,
     section?: string,
     breakpoint?: BREAKPOINT
-  ): SlotConfig {
-    const pageTemplateConfig = this.config.layoutSlots[templateUid];
+  ): SlotConfig | undefined {
```

Comment:
Why undefined instead of null?

Reply:
it can return `SlotConfig | undefined  | null`. `undefined` is returned by `if...else if... statement from line 195.

Reply:
ok